benchmark/benchmark.jl

```diff
@@ -57,7 +57,7 @@ end
 
 @info "initializing state"
 @time MocosSim.reset!(state, MersenneTwister(0))
-@time MocosSim.initialfeed!(state, 100)
+@time MocosSim.InstantOusideCases(;num_infections=100)(state, params)
```

Comment:
maybe you can correct InstantOusideCases to InstantOutsideCases in the code?

Reply:
this will break JSON files...again